### PR TITLE
Improve publish command output

### PR DESCRIPTION
**Pull request by digitalsadhu**
This PR aligns command output with some nice coloured formatting to match the formatting of the meta command

**Output for the package command:**

Screenshot 2020-05-06 at 21 43 01

**Output for the import map command:**

Screenshot 2020-05-06 at 22 51 58

**Output for the import npm command:**

Screenshot 2020-05-06 at 22 52 41

### CODE REVIEW

File: utils/version.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        this.type = type;
+        this.name = name;
+        this.org = org;
+        this.files = files;
+        this.meta = meta;
+    }
+
+    get version() {
+        return this[_version];
+    }
```
Comment (trygve-lie) [inline utils/version.js:53]: Are these getters and setters working? I thought they would collide with same properties on `this`. Does the `this.` values miss a `_`?

Comment (digitalsadhu) [reply utils/version.js:53]: not sure, will check.

**Comment by trygve-lie** [comment]
Looks nice.

Though; I think there is something wrong with the getters and setters in utils/version.js

File: utils/version.js (hunk 1 above)
Comment (trygve-lie) [reply utils/version.js:53]: I think there is something mixed up with your use of Symbols, this and setters / getters here.

Comment (digitalsadhu) [reply utils/version.js:53]: Looks like it works I think. In the constructor I set `this.version = version` which calls the setter function which in turn sets the version privately on `this[_version]`. Were you thinking that calling `this.version` in the constructor would override the setter? Or something else I'm missing here?

**Pull request merged**